src/lib.rs

```diff
@@ -131,28 +131,19 @@ pub struct Wkt<T>
 where
     T: WktFloat,
 {
-    pub items: Vec<Geometry<T>>,
+    pub item: Geometry<T>,
```

Comment:
`Wkt<T>(Geometry<T>)` might be better here? Not sure.

Reply:
No strong opinion from me, but I have a slight preference for how you have it.

Reply:
I think I'd prefer renaming `Geometry` to `Wkt` or an `into_geometry()` method.

Reply:
> No strong opinion from me, but I have a slight preference for how you have it.

Same, but do what you feel is best here. Where does the `into_geometry()` method come into it?

Reply:
> Where does the into_geometry() method come into it?

I meant `Wkt::into_geometry(self) -> Geometry`.

I'd say keep `item` for now and maybe change it in a follow-up PR if we figure out a better way.